Update player.py

durinov opened the pull request:
Add subtitles playback support without subssupport plugin only for dreambox,with help allow choosing subtitles in audio selection menu.

durinov committed:
Update player.py

Add subtitles playback support without subssupport plugin only for dreambox,with help allow choosing subtitles in audio selection menu.

skyjet18 commented:
Vďaka. Viem o tom (resp. predchádzajúcom) pull requeste a nezabudol som na neho. Len som ešte nemal čas na to pozrieť, resp. to otestovať.

skyjet18 commented:
Len pre upresnenie: Čo tento patch vlastne rieši? Píšeš, že pre DMM pridá podporu pre embedded titulky aj bez SubsSupport, no v kóde to máš spravené tak, že ak nie je nainštalovaný SubsSupport, tak to aj tak použije InfoBarSubSelectionNoSubtitles = audio menu bez titulkov. Trochu som ten patch na skúšku upravil a skúsil som to na DM One, no titulky mi bez SubsSupport nefungujú (aj keď ich v audio menu vyberiem).
Vďaka za info.

durinov committed:
Update player.py

Fatal error,when saved options config.plugins.archivCZSK.videoPlayer.subtitlesInAudioSelection (type=false),subtitle were not played are without subssupport plugin.

durinov committed:
Update player.py

skyjet18 commented:
Vďaka za nakopnutie s tým InfoBarSubsubtitleSupport. Ten tvoj patch som upravil a funguje to teraz na všetkých image vrátane dreamboxu. Rovnako všade už funguje aj výber titulkov v audio menu (vrátane VTi a to je už niečo). Pokiaľ je k dispozícii SubsSupport, tak sa použije ten. Ak nie, tak sa použije interná podpora pre embedded titulky priamo v image. Commit so zmenami je tu: https://github.com/archivczsk/archivczsk/commit/04a6db5eb9b7f761a999eeb00f55bfebdd7e17df a je to vydané von vo verzii 2.6.4.
Vzhľadom na úpravy pull request ako taký zatváram bez priameho mergnutia.

durinov commented:
Veľká vďaka.:-)Dňa Wed, 10 Apr 2024 17:20:39 +0000 (UTC) skyjet18  napísal:> Vďaka za nakopnutie s tým InfoBarSubsubtitleSupport. Ten tvoj patch >som upravil a funguje to teraz na všetkých image vrátane dreamboxu. >Rovnako všade už funguje aj výber titulkov v audio menu (vrátane VTi >a to je už niečo). Pokiaľ je k dispozícii SubsSupport, tak sa použije >ten. Ak nie, tak sa použije interná podpora pre embedded titulky >priamo v image. Commit so zmenami je tu: >https://github.com/archivczsk/archivczsk/commit/04a6db5eb9b7f761a999eeb00f55bfebdd7e17df >a je to vydané von vo verzii 2.6.4.> Vzhľadom na úpravy pull request ako taký zatváram bez priameho >mergnutia.> > -- > Reply to this email directly or view it on GitHub:> https://github.com/archivczsk/archivczsk/pull/18#issuecomment-2048092139> You are receiving this because you authored the thread.> > Message ID:
